### DISCUSSION TITLE
Use with spatie/laravel-medialibrary?

### DISCUSSION THREAD

**Discussion by juliomotol** [spatie/laravel-settings#50] [General]
Have anyone tried to use this with `spatie/laravel-medialibrary`? It would be great if we could attach media in to the settings property e.g. site icon, banners.

**Comment by hulkur** [comment]
Just ran into same problem. No obvious simple solution aside from overwriting constructor.

**Comment by judgej** [comment]
Did anyone have any ideas on how this could be done?
